_alerts/2017-07-12-potential-split.md

```diff
@@ -0,0 +1,106 @@
+---
+## This file is licensed under the MIT License (MIT) available on
+## http://opensource.org/licenses/MIT.
+
+title: "Potential network disruption"
+shorturl: "potential-split"
+active: true
+banner: "Warning: Bitcoin may be unsafe to use starting July 31st (click here for details)"
```

Comment:
Let's update the banner to match the title (I left a couple of suggestions in previous comments) and remove the "click here" as mentioned in the previous review.